### PR TITLE
SAK-29639 need better styles for disabled input elements

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-29639

These styles we removed when we went to Morpheus.

### CODE REVIEW

**Comment by bjones86** [comment]
@juanjmerono and @SedueRey, please feel free to review this. I tested it locally, works great.

File: reference/library/src/morpheus-master/sass/base/_defaults.scss [hunk 1]
```diff
@@ -107,3 +107,29 @@ u, ins {
...
+input[type="submit"][class="disabled"]:hover,
+input[type="button"][class="disabled"]:hover,
+button[disabled]:hover,
+button[class="disabled"]:hover,
+.formButtonDisabled, .formButtonDisabled:hover {
+	background: transparent linear-gradient(#EEE 0%, #EEE 100%) repeat scroll 0% 0% !important;
+	color: #BBB !important;
+	border: 1px solid #DDD !important;
+	box-shadow: none !important;
+	cursor: default;
```
Comment (SedueRey) [inline reference/library/src/morpheus-master/sass/base/_defaults.scss]: Just one thing, sorry if I didn't watch it yesterday, it was late yet on Murcia. The value for cursor should be 
`cursor: not-allowed;`

Comment (bjones86) [inline reference/library/src/morpheus-master/sass/base/_defaults.scss]: No problem, I'll fix this up right away.

**Comment by bjones86** [comment]
Fixed the cursor value.

**Comment by SedueRey** [comment]
Perfect :)

**Comment by bjones86** [comment]
Thanks to both of you for your help! I think it's safe to merge this when you're ready @juanjmerono

**Pull request merged**